src/QsCompiler/QirGeneration/Subtransformations/ExpressionKindTransformation.cs

```diff
@@ -544,7 +548,7 @@ Value TupleToString(TupleValue tuple)
                         }
                         else
                         {
-                            comma ??= CreateConstantString(",");
+                            comma ??= CreateConstantString(", ");
                             str = DoAppend(str, comma, unreferenceNext: false);
                         }
                     }
```

Comment:
FYI: Another way to construct a comma-separated list in parentheses is about like this:
```C++
// Output an opening parenthesis, or `$"{tuple.TypeName}("`.

for (var idx = 0; idx < tupleElements.Length; ++idx)
{
    if(idx != 0)
    {
        // Output comma (`, `).
    }
    // Output `element[i]`.
}

// Output the closing parenthesis `)`.
```

Reply:
Sure, either way is fine for me, so I changed it.